documentation/RLC-test-quickstart.md

```diff
@@ -0,0 +1,234 @@
+- Step 2: Add the utility method `createClient` in `public/utils/recordedClient.ts` to share the `PurviewCatalogClient` creation.
+  - Call `createTestCredential` to init your credential and refer [here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md#aad-and-the-new-noopcredential) for more details
+  - Wrap the `option` with test options by calling `recorder.configureClientOptions(options)`
+- Step 3: Call `createClient` to prepare the client and call `client.path("/atlas/v2/glossary").post()` to create our glossary resource
+- Step 4: Specify environment variables that would be replaced in the recording and set during playback in the map `envSetupForPlayback` under the file `public/utils/recordedClient.ts`. This could be used to ensure that secrets and user-specific options do not appear in the recording body
+- Step 5: Add necessary assertions in your test case, then run and record your test
+
+`glossary.spec.ts`
+
+```typescript
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { PurviewCatalogClient } from "../../src";
+import { createClient, createRecorder } from "./utils/recordedClient";
+
+describe("My test", () => {
+  let recorder: Recorder;
+  let client: PurviewCatalogClient;
+  let glossaryName: string;
+
+  beforeEach(async function () {
+    recorder = await createRecorder(this);
+    client = await createClient(recorder);
+    glossaryName = "js-testing";
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Should create a glossary", async () => {
+    const glossary = await client.path("/atlas/v2/glossary").post({
+      body: {
+        name: glossaryName,
+        shortDescription: "Example Short Description",
+        longDescription: "Example Long Description",
+        language: "en",
+        usage: "Example Glossary",
+      },
+    });
+    assert.strictEqual(glossary.status, "200");
+  });
+});
+```
+
+`utils/recordedClient.ts`
+
+```typescript
+import { Context } from "mocha";
+import { Recorder, RecorderStartOptions } from "@azure-tools/test-recorder";
+import PurviewCatalog, { PurviewCatalogClient } from "../../../src";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { ClientOptions } from "@azure-rest/core-client";
+import "./env";
+
+const envSetupForPlayback: Record<string, string> = {
+  ENDPOINT: "https://endpoint",
+  AZURE_CLIENT_ID: "azure_client_id",
+  AZURE_CLIENT_SECRET: "azure_client_secret",
+  AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
+  SUBSCRIPTION_ID: "azure_subscription_id",
+  PURVIEW_CATALOG_GLOSSARY_ENV: "glossary_custom_env", // Add environment vara you'd like to replace
+};
+
+const recorderEnvSetup: RecorderStartOptions = {
+  envSetupForPlaayback,
+};
+
+/**
+ * creates the recorder and reads the environment variables from the `.env` file.
```

Comment:
what is the location of .env file and what information shall i put into it?

Reply:
Will add this part